src/components/questions/ImageUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  ToastAndroid,
+  Alert,
+  PermissionsAndroid,
+} from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+const ImageUpload = () => {
```

Comment:
Hoe krijgen we de foto nu eruit?

Reply:
ik heb hiervoor een remove functie erbij gezet waarbij een lege SetImage word gestuurd

Reply:
Oke dat is goed, maar hoe kan ik deze opsturen naar de API?